tests/e2e/matmul/BUILD.bazel

```diff
@@ -38,7 +38,7 @@ py_binary(
 [iree_generated_trace_runner_test(
     name = "e2e_matmul_mmt4d_%s_small" % lhs_rhs_type,
     compiler_flags = [
-        "--iree-flow-enable-data-tiling",
+        "--iree-global-opt-enable-data-tiling",
```

Comment:
have been meaning to mention it for awhile but now you're touching a flag so it's a good time :) let's start dropping `enable` from flag names - when flags are boolean what decides if something is enabled or not is the value, not the presence of the flag, and it's just an extra word (so `--iree-global-opt-data-tiling=true` instead of `--iree-global-opt-enable-data-tiling=true` which is redundant)